### ISSUE TITLE
Postgress min version in docs

### ISSUE THREAD

**Issue by mllocs** [PopulateTools/gobierto#3425]
## TLDR

This is just an edit suggestion in the gobierto docs regarding the minimal postgres version. 
It seems to be >=12 instead of >=9.6.

## What

I've been setting up gobierto in my local environment following this documentation: https://gobierto.readme.io/docs/development-environment-1

I tried running the app with `postgres@10` but it didn't work. I did it because it was the version I already had installed. I was getting this error: 

```
ERROR:  unrecognized configuration parameter "default_table_access_method"
```

After looking into it, it looks like that configuration parameter that gobierto uses was only added in `postgres@12`. The docs say that gobierto is compatible with `>= postgres@9.6` but maybe at some point something changed and the docs became outdated. 

That's all, I didn't suggest an edit in readme.io because I couldn't add any context to it 😄

**Comment by ferblape** [comment]
Yes, we changed this dependency a few weeks ago: to implement the new full text search we use a feature of Postgres 12. I'll update the documentation ASAP, thanks.
